spec/lib/rmt/cli/repos_spec.rb

```diff
@@ -87,9 +87,9 @@
       let(:argv) { ['disable', repository.scc_id.to_s, repository_2.scc_id.to_s, repository_3.scc_id.to_s] }
       let(:expected_output) do
         <<-OUTPUT
-Repository by id #{repository.scc_id} successfully disabled.
-Repository by id #{repository_2.scc_id} successfully disabled.
-Repository by id #{repository_3.scc_id} successfully disabled.
+Repository by ID #{repository.scc_id} successfully disabled.
```

Comment:
This is inconsistent with other places, where the ID is also put in single-quotes. Same file, line 56 for example.
I think, we shouldn't quote IDs.